RiotSharp/TournamentRiotApi.cs

```diff
@@ -69,157 +69,117 @@ internal static TournamentRiotApi GetInstance()
             }
             return instance;
         }
-    
-        public TournamentProvider CreateProvider(Region region, string url)
+
+        #region (Current) Version 3
```

Comment:
I think we should keep that region. It makes the code more readable. I'm just gonna rename it to `Public Methods`